microsoft/c2pa-extension-validator
# Adds TSA cert trust list

Fixes issue #46.

The c2pa SDK doesn't yet accept TSA trust lists. We need to manually verify that the TSA cert from the `tstToken` match a cert in the TSA trust list added by this PR.

To test:
* Add the [test-trust-list](https://github.com/microsoft/c2pa-extension-validator/blob/main/test/test-trust-list.json) and the (new) [trusted TSA cert](https://github.com/microsoft/c2pa-extension-validator/blob/tsa-trust-store/test/trustedTSA/digicert_TSA_CA.pem)
* Visit the [unit test page](https://github.com/microsoft/c2pa-extension-validator/blob/main/test/unit-tests.html). The "Valid: expired cert, time-stamped by trusted TSA" asset should be valid, the "Invalid: expired cert, time-stamped by untrusted TSA" should be invalid.
